### ISSUE TITLE
ini_manage should support different separator types

### ISSUE THREAD

**Issue by boltronics** [saltstack/salt#31209]
I am configuring Munin in a custom state file, and Munin basically uses an ini format for the config file format. The only difference is that instead of separating key and value pairs with an equal sign ("="), it just uses a space.

It would be great if ini_manage supported a "separator" argument, so the built-in state and execution modules could be reused for these kind of files.

**Comment by rallytime** [comment]
Hi @boltronics. I see your implementation of this feature has been merged into develop. Can this issue be closed or does more work need to be done first?

**Comment by boltronics** [comment]
Hi @rallytime. I'm happy with the patch, however IIRC from reading other Salt issue pages, being on the develop branch doesn't automatically mean the code will hit a release milestone, as commits are cherry-picked. Can you please clarify this?

If it will end up in a future release, I'm happy for this to be closed. Thanks.

**Comment by rallytime** [comment]
@boltronics Since this was a feature addition, you correctly submitted the change against the develop branch. We cut new feature release branches from develop. Since this change went into the develop branch after we cut the 2016.3 branch, your feature addition will be in the next feature release (code named Carbon). 

That way the new feature can be vetted, tested, and adjusted on the develop branch in the next months until it is time to cut a new feature release. Does that help explain the process? There some more docs about that [here](https://docs.saltstack.com/en/latest/topics/development/contributing.html#which-salt-branch) if you'd like to learn more.

**Comment by boltronics** [comment]
Brilliant! Thanks for the explanation.
